macOS/DuckDuckGo/Feedback/View/FeedbackViewController.swift

```diff
@@ -283,13 +285,13 @@ final class FeedbackViewController: NSViewController {
     }
 
     var isOsUnsupported: Bool {
-        return !SupportedOSChecker.isCurrentOSReceivingUpdates
+        return !supportedOSChecker.isCurrentOSReceivingUpdates
     }
 
     private weak var unsupportedOsChildView: NSView?
     private func showUnsupportedOsViewIfNeeded() {
         if isOsUnsupported && unsupportedOsChildView == nil {
-            let view = NSHostingView(rootView: Preferences.UnsupportedDeviceInfoBox(wide: false))
+            let view = NSHostingView(rootView: Preferences.UnsupportedDeviceInfoBox(wide: false, supportedOSChecker: supportedOSChecker))
```

Comment:
One of the good thing about this change is we can now pass the checker along, and make this whole thing slightly more testable.